README.md

```diff
@@ -100,6 +100,7 @@ Finally, enable all of the rules that you would like to use.  Use [our preset](#
 * [react/no-redundant-should-component-update](docs/rules/no-redundant-should-component-update.md): Prevent usage of `shouldComponentUpdate` when extending React.PureComponent
 * [react/no-render-return-value](docs/rules/no-render-return-value.md): Prevent usage of the return value of `React.render`
 * [react/no-set-state](docs/rules/no-set-state.md): Prevent usage of `setState`
+* [react/no-static-typos](docs/rules/no-static-typos.md): Prevent casing typos when declaring static class properties.
```

Comment:
In addition to casing, it would be nice to flag also anything with a small levenshtein distance. That could be done as a separate PR.

And, if you are interested, it would be really cool to have a similar rule for detecting typos in lifecycle methods: https://github.com/yannickcr/eslint-plugin-react/issues/843

I wonder if this rule should have a more general name like `react/no-typos` and make it work also for lifecycle methods?